src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1896,6 +1896,83 @@ private with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
+        update contact's address override to an address in another account
```

Comment:
Does not including the column of * at the start of each line still work with ApexDoc?